External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -414,27 +416,42 @@ public static void HandleAddClosingBraces(ScintillaControl sci, char c, bool add
                         HandleRemoveBrace(sci, c, braces);
                 }
             }
+            // If the current style is a string literal in double/single quotes and the matching char is entered
+            else if (c == '"' && IsDoubleQuoteStyle(style) || c == '\'' && IsSingleQuoteStyle(style))
+            {
+                // If the next char is the corresponding closing char
+                if (addedChar && c == sci.CurrentChar)
```

Comment:
Just like Visual Studio, the closing quote still is overridden. I would want it to not override for escaped quotes, but I figured that the process can get complicated. One case I could think of was when the leading backslash is also an escaped backslash, and in the process of checking if there is a backslash before the backslash, gotta check if it is escaped...